admin/class-admin-init.php

```diff
@@ -311,6 +312,57 @@ private function get_compatibility_notification() {
 	}
 
 	/**
+	 * Add an alert if outdated versions of Yoast SEO plugins are running.
+	 */
+	public function yoast_plugin_compatibility_notification() {
+		$compatibility_checker = new WPSEO_Plugin_Compatibility( WPSEO_VERSION );
+		$plugins = $compatibility_checker->get_installed_plugins_compatibility();
+
+		$notification_center = Yoast_Notification_Center::get();
+
+		foreach ( $plugins as $name => $plugin ) {
+
+			if ( $plugin['compatible'] === false && $plugin['active'] === true ) {
+				$notification = $this->get_yoast_seo_compatibility_notification( $plugin, Yoast_Notification::ERROR );
```

Comment:
This can be simplified to:

```php
if ( $plugin['compatible'] === false ) {
    $type = $plugin['active'] ? Yoast_Notification::ERROR : Yoast_Notification::WARNING;
    $notification = $this->get_yoast_seo_compatibility_notification( $plugin, $type );

    $notification_center->add_notification( $notification );
}
```